omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitAttributeResource1_9.java

```diff
@@ -95,7 +95,7 @@ public VisitAttribute getByUniqueId(String uniqueId) {
 	 */
 	@Override
 	public VisitAttribute save(VisitAttribute delegate) {
-		if (delegate.getAttributeType().getMaxOccurs() == 1) {
+		if (delegate.getAttributeType().getMaxOccurs() == 1 && delegate.getAttributeType().getMaxOccurs() != null) {
```

Comment:
@Bisht13 did you get a chance to test this and confirm it works?

Reply:
No @dkayiwa I haven't tested it. Working on it. 😃